web/src/test/java/com/graphhopper/http/BaseServletTester.java

```diff
@@ -109,6 +109,10 @@ private void bootJetty(CmdArgs args, int retryCount) {
         }
     }
 
+    GHServer getServer(CmdArgs args) {
```

Comment:
I think this is not necessary as you can call BaseServletTester.setUpGuice(modules)

Reply:
Here I meant that you can customize the modules etc via extending the modules, there is no need to customize GHServer, nor to create this method. I think you can search for TestModule and how to utilize this (this would then also better work with dropwizard IMO)

Reply:
Ah thanks. I think I now understood :)

Reply:
I think, there is no TestModule. Probably have created it in another project instead of calling `setUpGuice(server.createModule());`